onvm/onvm_nflib/onvm_pkt_common.c

```diff
@@ -102,9 +102,8 @@ onvm_pkt_process_tx_batch(struct queue_mgr *tx_mgr, struct rte_mbuf *pkts[], uin
                         // and !<return value> is 1.
                         nf->stats.act_drop++;
                         nf->stats.tx += !onvm_pkt_drop(pkts[i]);
-                } else if (meta->action == ONVM_NF_ACTION_NEXT) {
-                        /* TODO: Here we drop the packet : there will be a flow table
-                        in the future to know what to do with the packet next */
+                } else if (meta->action == ONVM_NF_ACTION_NEXT || meta->destination == ACTION_NEXT_DEST_ID) {
```

Comment:
@bdevierno1 setting 255 as the next action would definitely decrease performance, but I think that (from CI) we see that just having to process this or `||` statement in the condition slows down other NFs. Because we know that speed_tester would not have set the `meta->destination` to 255. I think that's the cause of the ~4% performance drop from CI

Reply:
Yeah this is correct. Unfortunately, I tried different ways to optimize this but did not see any substantial improvements. By removing the OR I will still have to set another conditional which will have to be placed before this statement so as to maintain the same logic. Perhaps there is another way to do this?

Reply:
Instead of doing meta->action == 255, can we define a new action that denotes this is a packet meant for flow rule lookup?